Samples/FlapOrDie/FlapOrDie.csproj

```diff
@@ -1,75 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+	</PropertyGroup>
+
   <PropertyGroup>
-    <DualityPackageExcludeParentElement />
-    <StartAction>Program</StartAction>
-    <StartProgram>$(SolutionDir)Build\Output\DualityEditor.exe</StartProgram>
-    <StartWorkingDirectory>..\</StartWorkingDirectory>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
I think this isn't needed as well. With .net sdk these properties are part of the csproj itself instead of a separate assembly.cs file

Reply:
Example: https://github.com/Barsonax/Singularity/blob/master/src/Singularity/Singularity.csproj#L6